### PR TITLE
Fix aliases

### PR DESCRIPTION
**Pull request by ibgreen**
Setting them in the local onCreateWebpackConfig override works for some reason...

### CODE REVIEW

File: website/src/components/animation-loop-runner.jsx [hunk 1]
```diff
@@ -74,7 +79,7 @@ export default class AnimationLoopRunner extends Component {
 
   componentWillUnmount() {
     this.animationLoop.stop(this.props);
-    this.animationLoop.finalize();
+    this.animationLoop.delete();
```
Comment (xintongxia) [inline website/src/components/animation-loop-runner.jsx]: What's the issue with `finalize()`?

File: website/src/components/animation-loop-runner.jsx [hunk 2]
```diff
@@ -55,12 +55,17 @@ export default class AnimationLoopRunner extends Component {
   componentDidMount() {
     const {showStats} = this.props;
 
+    console.log(this.props);
```
Comment (xintongxia) [inline website/src/components/animation-loop-runner.jsx]: lint doesn't apply to this file? Delete `console`

File: website/src/components/animation-loop-runner.jsx (hunk 1 above)
Comment (ibgreen) [reply website/src/components/animation-loop-runner.jsx]: > What's the issue with finalize()?

For historic reasons, deck.gl uses `finalize()`. luma.gl uses `delete()`.

File: website/package.json [hunk 3]
```diff
@@ -29,10 +29,11 @@
     "react-dom": "^16.6.0"
   },
   "devDependencies": {
+    "eslint-import-resolver-alias": "^1.1.2",
```
Comment (xintongxia) [inline website/package.json]: Is this being used?

Comment (ibgreen) [reply website/package.json]: Good catch will remove in next PR.

**Pull request merged**